motorway/contrib/amazon_sqs/utils.py

```diff
@@ -1,10 +1,17 @@
 import json
 from boto.sqs.message import Message
+from motorway.contrib.amazon_sqs.exceptions import SQSMessageToLarge
+
+SQS_MESSAGE_MAXIMUM_SIZE = 262144
 
 
 class SQSJSONMessage(Message):
     def encode(self, value):
-        return json.dumps(value)
+        s = json.dumps(value)
+        message_size = len(s.encode('utf-8'))
+        if message_size > SQS_MESSAGE_MAXIMUM_SIZE:
+            raise SQSMessageToLarge('message was %s in bytes and cant be bigger than %s' % (message_size, SQS_MESSAGE_MAXIMUM_SIZE))
```

Comment:
``'Message size of %sB exceeds the maximum SQS message size of %sB'``

Reply:
fixed